app/models/concerns/suppressible.rb

```diff
@@ -0,0 +1,17 @@
+module Suppressible
+  def self.registry
+    ActiveSupport::IsolatedExecutionState[:suppressible_registry] ||= {}
```

Comment:
This is interesting! we can chat about this later

Reply:
Ah, forgot to explain that - updated the description accordingly.